src/js/claims-status/containers/AppealInfo.jsx

```diff
@@ -6,73 +6,97 @@ import { Link } from 'react-router';
-        </Breadcrumbs>
-      </div>
-      <div className="row">
-        <h1>{claimHeading}</h1>
-      </div>
-      <div className="row">
-        <div className="medium-8 columns">
-          <AppealsV2TabNav
-            appealId={appealId}/>
-          <div className="va-tab-content va-appeals-content">
-            {React.Children.map(children, child => React.cloneElement(child, { appeal }))}
+  render() {
+    const { params, appeal, appealsLoading, children } = this.props;
+    let appealContent;
+    if (appeal) {
+      const claimHeading = this.createHeading();
+      appealContent = (
+        <div>
+          <div className="row">
+            <Breadcrumbs>
+              <li><Link to="your-claims">Your Claims and Appeals</Link></li>
+              <li><strong>{claimHeading}</strong></li>
+            </Breadcrumbs>
+          </div>
+          <div className="row">
+            <AppealHeader
+              heading={claimHeading}
+              lastUpdated={appeal.attributes.updated}/>
+          </div>
+          <div className="row">
+            <div className="medium-8 columns">
+              <AppealsV2TabNav
+                appealId={params.id}/>
+              <div className="va-tab-content va-appeals-content">
+                {React.Children.map(children, child => React.cloneElement(child, { appeal }))}
+              </div>
+            </div>
+            <div className="medium-4 columns help-sidebar">
+              {appeal && <AppealHelpSidebar location={appeal.attributes.location} aoj={appeal.attributes.aoj}/>}
+            </div>
           </div>
         </div>
-        <div className="medium-4 columns help-sidebar">
-          {appeal && <AppealHelpSidebar location={appeal.attributes.location} aoj={appeal.attributes.aoj}/>}
-        </div>
-      </div>
-    </div>
-  );
+      );
+    } else if (appealsLoading) {
+      appealContent = <LoadingIndicator message="Please wait while we load your appeal..."/>;
+    } else {
+      // Appeals finished loading but appeal not found, so the ID passed in the params
+      // doesn't match any appeals in Redux appeals array (at least not for this user)
+      // TO-DO: Get input from content team
+      appealContent = <AppealNotFound/>;
+    }
+
+    return (appealContent);
+  }
 }
 
 AppealInfo.propTypes = {
   params: PropTypes.shape({ id: PropTypes.string.isRequired }).isRequired,
-  children: PropTypes.object.isRequired
+  children: PropTypes.node.isRequired,
+  appealsLoading: PropTypes.bool.isRequired,
+  appeal: PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    type: PropTypes.string.isRequired,
+    attributes: PropTypes.object.isRequired,
+  })
 };
 
 function mapStateToProps(state, ownProps) {
+  const { appealsLoading } = state.disability.status.appeals;
+  // appealsLoading is not initialized in Redux, it's added once fetch starts. We
+  // need it to be available immediately to know whether to show the loading spinner
```

Comment:
I thought about this but wasn't sure if it would have negative consequences for v1. Reason I thought it might is that `available` is actually explicitly initialized to `true`, so whoever initially made the reducers seems to have left `appealsLoading` uninitialized on purpose. I didn't really dig into it more than that...maybe I could throw a comment in here to revisit once v2 is launched, just to simplify things.